x/treasury/spec/05_events.md

```diff
@@ -0,0 +1,26 @@
+# Events
+
+The bonds module emits the following events:
+
+## EndBlocker
```

Comment:
All events were placed under "EndBlocker" title, but none of the events are generated from the EndBlocker. The events are supposed to be categorised based on the message or function from where they get generated.

Reply:
removed all from endblocker and added in Handler

Reply:
Please remove extra title (from table of contents as well please)